manifest.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Is this just for reference?

Reply:
it's what I was using for local testing and I didn't mean to include it. It could serve as a sample, I suppose